badfish/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM quay.io/quads/badfish
+COPY bad-hosts /
```

Comment:
Please change the name from `bad-hosts` to `dell-hosts` or something similar. bad-hosts is a misleading name :).

Reply:
I took this term from badfish repository, in which they have used it for an example.

Reply:
`bad-hosts` sounds like it's referring to hosts which are faulty/bad. `dell-hosts` or something else like `bf-hosts` would be more appropriate.

Reply:
okay. will update it to 'dell-hosts'.